tests/load_test/normal_loadtest.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+from load_test.models import Payload, OrdinaryLoadTestAgent
+from pathlib import Path
+import time
+import click
+
+
+@click.command()
+@click.option('--counts', default=1, help='Number of notifications.')
+@click.option('--url', help='The url to the endpoint of Lira you want to test aginst. e.g http://pipelines.dev.data.humancellatlas.org/notifications')
+@click.option('--auth_token', help='The value of auth token to the Lira, could be set to False to disable it.')
+@click.option('--payload_file', help='Path to the notification file. e.g. ./data/notification/notification.json')
+@click.option('--result_folder', help='Path to the result folder. e.g. ./data/results/')
+@click.option('--lira_version', help='The version of Lira to be tested against. e.g. v0.7.0')
+@click.option('--environment', help='The running environment of Lira. gunicorn/docker/gke')
+@click.option('--mode', help='The running mode of Lira, dry_run/live_run')
+@click.option('--cromwell_type', help='The type of Cromwell which the Lira is connecting to. null/instance/caas')
+@click.option('--caching', help='The caching switch of Lira. True/False')
+def normal_loadtest(counts, url, auth_token, payload_file, result_folder, lira_version, environment, mode,
+                    cromwell_type, caching):
+    destination = Path(result_folder) / 'load_test_result_{}.json'.format(time.strftime("%Y%m%d-%H%M%S"))
+    scenario = {
+        'lira': lira_version,
+        'environment': environment,
+        'mode': mode,
+        'cromwell': cromwell_type,
+        'caching': caching,
+    }
+
+    if auth_token:
+        auth_token = {'auth': auth_token}
+    else:
+        auth_token = None
+    # FIXME: Using https in url here will run into a "RecursionError: maximum recursion depth exceeded" bug of requests
```

Comment:
Oh no! Should we be concerned about sending the auth token using `http`?

Reply:
I think http here is not that unsafe. Besides, as we already have plans to replace tokens with the a new Auth approach, I think we are good? I will file an issue to the requests repo tho.

Reply:
I'm not sure, but maybe @dshiga has some insight here?

Reply:
It's okay to use http when running Lira on a laptop but if running against a deployed Lira it should be https.

Reply:
ok, thanks for the clarification!

Reply:
Is there really no way to make this work with https? Maybe it would be best to add a check so that if you try to use it against anything but localhost it will raise an error.

Reply:
I managed to solve the problem.